libs/qCC_glWindow/ccGLWindow.cpp

```diff
@@ -4248,6 +4256,14 @@ void ccGLWindow::mouseMoveEvent(QMouseEvent *event)
 			}
 		}
 	}
+	else if ((event->buttons() & Qt::MiddleButton)) // zoom
+	{
+		//middle button = zooming
+		float pseudo_wheelDelta_deg = -dy;
```

Comment:
Isn't this producing a warning about type conversion ? Can you maybe use static_cast<float>(-dy)?

Reply:
did not warn for me (qtcreators clang-tidy and clazy warnings), but added it anyways